Assets/Scripts/Achievements/IBulletHitEnemyAchivement.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public interface IBulletHitEnemyAchivement 
```

Comment:
Instead of this interface , you can use observer pattern , to check when an event is happening and invoke them when it is needed.